cmd/gosky/admin.go

```diff
@@ -364,6 +366,13 @@ var listReportsCmd = &cli.Command{
 		&cli.BoolFlag{
 			Name: "raw",
 		},
+		&cli.BoolFlag{
+			Name:  "resolved",
+			Value: true,
+		},
+		&cli.BoolFlag{
+			Name: "template-output",
```

Comment:
was this added accidentally? don't see it used (at least in the PR diff)

Reply:
Yeah, some other WIP progress got rolled up here by accident. Was very carefully avoiding pulling it in until i wasnt..